lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/NodeDAOImplTest.java

```diff
@@ -2339,80 +2340,35 @@ public void testCreateTableNode() throws DatastoreException, InvalidModelExcepti
 		assertEquals(expected, n1.getColumnModelIds());
 	}
 
+	/**
+	 * PLFM-5960
+	 * @throws Exception
+	 */
 	@Test
 	public void testGetEntityHeaderByMd5() throws Exception {
 
 		// Nothing yet
-		List<EntityHeader> results = nodeDao.getEntityHeaderByMd5("md5");
+		List<EntityHeader> results = nodeDao.getEntityHeaderByMd5(fileHandle3.getContentMd5());
 		assertNotNull(results);
 		assertEquals(0, results.size());
 
-		// Add a node with a file handle
-		Node node1 = NodeTestUtils.createNew("testGetEntityHeaderByMd5 node 1", creatorUserGroupId);
-		node1.setFileHandleId(fileHandle.getId());
-		final String node1Label1 = "1";
-		node1.setVersionLabel(node1Label1);
-		final String id1 = nodeDao.createNew(node1);
-		node1 = nodeDao.getNode(id1);
-		assertNotNull(id1);
-		toDelete.add(id1);
-		node1.setId(id1);
-
-		results = nodeDao.getEntityHeaderByMd5(fileHandle.getContentMd5());
-		assertNotNull(results);
-		assertEquals(1, results.size());
-		assertEquals(id1, results.get(0).getId());
-		assertNotNull(results.get(0).getBenefactorId());
-		assertEquals(Long.valueOf(1L), results.get(0).getVersionNumber());
-		assertEquals(node1Label1, results.get(0).getVersionLabel());
-		assertEquals(node1.getCreatedByPrincipalId().toString(), results.get(0).getCreatedBy());
-		assertEquals(node1.getCreatedOn(), results.get(0).getCreatedOn());
-		assertEquals(node1.getModifiedByPrincipalId().toString(), results.get(0).getModifiedBy());
-		assertEquals(node1.getModifiedOn(), results.get(0).getModifiedOn());
-
-		// Create a new version of the node of the same file
-		final String node1Label2 = "Node 1 version label 2";
-		node1.setVersionLabel(node1Label2);
-		nodeDao.createNewVersion(node1);
-
-		results = nodeDao.getEntityHeaderByMd5(fileHandle.getContentMd5());
-		assertNotNull(results);
-		assertEquals(2, results.size());
-		assertEquals(id1, results.get(0).getId());
-		assertEquals(id1, results.get(1).getId());
-		assertFalse(results.get(0).getVersionNumber().equals(results.get(1).getVersionNumber()));
-
-		// Add a new node with no file handle
-		Node node2 = NodeTestUtils.createNew("testGetEntityHeaderByMd5 node 2", creatorUserGroupId);
-		final String node2Label1 = "Node 2 version label 1";
-		node1.setVersionLabel(node2Label1);
-		final String id2 = nodeDao.createNew(node2);
-		assertNotNull(id2);
-		toDelete.add(id2);
-		node2.setId(id2);
+		for(int i = 0; i < 202; i++) {

```

Comment:
Let us separate this into its own test, e.g. testGetEntityHeaderByMd5WithOver200Macthing